### ISSUE TITLE
Way of downloading a log of the result - option to include a list of pairings

### ISSUE THREAD

**Issue by xtbqimuxmvtx** [arcanis/secretsanta#28]
I've used this generator extensively in November the last two years, and found it super helpful, but in order to do testing easier I've had to open every single url and hit enter to confirm I want to open for each member of my family (13 this year). I also would like to keep a record that's offline, that I can't see; so I can double check if there's confusion or someone loses their link and not be dependent on a specific host to have a copy online (although I guess running my own and as long as I have the urls I should be able to extract and use the key?)

TL;DR: 
* Download log (Includes spoilers!) button

**Comment by arcanis** [comment]
You can now download a CSV of the generated pairings

**Comment by christopher-nash** [comment]
When I generate our families list, since I also don't want any of the surprises spoiled for me, I just copy the table of created links and paste them into a spreadsheet. Then when someone inevitably says "I don't remember who I got a f I can't find the link" I just send them a copy of the original link, keeping all secrets and spoilers intact for me as the host.

**Comment by xtbqimuxmvtx** [comment]
Yeah I did the same this year but had trouble copying the table, (I may not have tried very hard) since the urls were cut off. I think zooming out to 10% worked fine. But the main gain is next year I will blacklist who had who this year, so a log that I don’t have to click on every user was reason for this issue.
